Sdtrig.tex

```diff
@@ -319,6 +319,32 @@ \subsection{Cache Operations}
     \end{steps}
 \end{commentary}
 
+\subsection{Invalid Virtual Addresses}
+
+\begin{steps}{For address matches without a mask, \RcsrTdataTwo must be able to:}
+    \item Hold all valid physical addresses (which are zero extended).
+    \item Hold all valid virtual addresses (which are sign extended).
+\end{steps}
```

Comment:
It only needs to support these sign-extended virtual addresses if Sv* is supported.  Also, the sign extension is a RV64 thing.  An M-mode-only RV32 implementation only needs to be able to hold PAs.  A RV32 implementation with Sv32 needs 32 flops and there is no sign extension.

Could this be simplified to say that it needs to be able to hold all valid addresses in all translation modes?  Then maybe some non-normative thing that explains that this means all PAs and all VAs in all supported Sv* modes.

Reply:
I've restructured this like you suggested. Not quite sure about the commentary I wrote.